runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -120,6 +106,13 @@
 
         /** Sets expireAfterAccess and expireAfterWrite in seconds. */
         @Default long cacheExpiryTime = Long.MAX_VALUE;
+
+        /** Sets the period of retrieving the latest trim mark in minutes. */
+        @Default long trimMarkSyncPeriod = 1;
+
+        /** Sets the period of trimming the resolvedQueues in minutes. **/
+        // FIXME: Remove this with the Stream Layer refactor.
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)